publish.html

```diff
@@ -134,93 +109,59 @@ <h3>OMERO.web</h3>
 <div class="line"><br /></div> <!-- end #line -->
 </div> <!-- end #line-block -->
 
-<img alt="../images/publishSplitWeb.jpg" class="align-center" src="images/publishSplitWeb.jpg" style="width: 100%;" />
+<img alt="../images/publishDropdownWeb.jpg" class="align-center" src="images/publishDropdownWeb.jpg" style="width: 82%;" />
 
 <div class="line-block">
 <div class="line"><br /></div> <!-- end #line -->
 </div> <!-- end #line-block -->
 
 </li>
 
-<p class="top_link"><a href="#top"><img src="images/back_to_top.png" alt="Image for link to top of page" style="float: right;"/></a></p>
-
-<div class="line-block">
-<div class="line"><br /></div> <!-- end #line -->
-</div> <!-- end #line-block -->
-
-<li><p class="first">Select <span class="bold">ROI Split Figure...</span> to create a split figure of the area of an image marked by a rectangular ROI.</p>
 
-<p>The full image thumbnail indicates the area of the image selected by the ROI.</p>
-
-<h3>OMERO.insight</h3>
-
-<div class="line-block">
-<div class="line"><br /></div> <!-- end #line -->
-</div> <!-- end #line-block -->
-
-<img alt="../images/publishROISplitInsight.jpg" class="align-left" src="images/publishROISplitInsight.jpg" style="width: 75%;" />
-
-<div class="line-block">
-<div class="line"><br /></div> <!-- end #line -->
-</div> <!-- end #line-block -->
-
-</li>
-
-<p class="top_link"><a href="#top"><img src="images/back_to_top.png" alt="Image for link to top of page" style="float: right;"/></a></p>
+<li><p class="first">Each script will open in a custom window.</p>
```

Comment:
I think that this whole section (including the publishAll.jpg) feels out of place now in the "Creating movies" workflow since most of the info refers to other scripts and the screenshot is of other scripts gets in the way of the Creating movies workflow.

It could simply be removed entirely (it doesn't contain much info that isn't covered elsewhere), or it could be moved to the equivalent spot under "Using scripts to create figures".